src/components/Request/NewRequestForm.Wizard.tsx

```diff
@@ -0,0 +1,193 @@
+import { useReducer } from "react";
+import { useFormContext } from "react-hook-form";
+import { Button, Spinner } from "@fluentui/react-components";
+import HiringInfo from "./NewFormSection/NewForm.HiringInfo";
+import JOA from "./NewFormSection/NewForm.JOA";
+import JobBoard from "./NewFormSection/NewForm.JobBoard";
+import LinkedInPost from "./NewFormSection/NewForm.LinkedInPost";
+import LinkedInSearch from "./NewFormSection/NewForm.LinkedInSearch";
+import RoutingInfo from "./NewFormSection/NewForm.Routing";
+import USAJobs from "./NewFormSection/NewForm.USAJobs";
+import Done from "./NewFormSection/NewForm.Done";
+
+interface INPUTSTEPFUNCTIONS {
+  next: (_methods: string[]) => string;
+  prev: (_methods: string[]) => string;
+}
+
+const INPUTSTEPS = new Map<string, INPUTSTEPFUNCTIONS>([
+  [
+    "RoutingInfo",
+    {
+      next: (_methods: string[]) => "HiringInfo",
+      prev: (_methods: string[]) => "RoutingInfo",
+    },
+  ],
+]);
+
+INPUTSTEPS.set("HiringInfo", {
+  next: (methods: string[]) => {
```

Comment:
No reason to type all the methods as array of strings throughout this file, as you already defined it in the interface.